packages/expo-calendar/ios/Conversions/EnumConversions.swift

```diff
@@ -0,0 +1,139 @@
+import ExpoModulesCore
+import EventKit
+
+func eventAvailability(_ availability: EKEventAvailability) -> String {
```

Comment:
`eventAvailability` as a function name doesn't tell too much about what it does. What would you say about `eventAvailabilityToString`?
The same applies to other functions here.